resources/instructions/landingPageTemplateChoice.html

```diff
@@ -0,0 +1,262 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Offline Starter Kit: Select Landing Page</title>
+        <style>
+            .landing-page-option-set {
+                display: flex;
+                flex-direction: row;
+                flex-wrap: nowrap;
+            }
+            .landing-page-option-title {
+                margin: 0px;
+                font-weight: bold;
+                font-size: 12px;
+            }
+            .landing-page-option-subtitle {
+                margin: 0px;
+                font-size: 10px;
+            }
+            h1 {
+                font-size: 18px;
+            }
+            .warning,
+            .error {
+                color: #f00;
+                font-size: 12px;
+                margin: 3px 5px;
+            }
+            .enabled-content-toggle.disabled {
+                opacity: 0.25;
+            }
+            #globalError,
+            #chooseTemplateButton {
+                margin-top: 12px;
+                margin-bottom: 12px;
+            }
+        </style>
+    </head>
+    <body>
+        <h1>Select a Landing Page Template</h1>
+
+        <p id="globalError" class="error" style="display: none"></p>
+        <div id="landingPageCollection" class="enabled-content-toggle">
+            <label id="existingLandingPageTemplateLabel">
+                <div class="enabled-content-toggle landing-page-option-set">
+                    <input
+                        type="radio"
+                        id="existingLandingPage"
+                        name="landingPageOption"
+                        value="existing"
+                    />
+                    <div>
+                        <p class="landing-page-option-title">Keep Existing</p>
+                        <p class="landing-page-option-subtitle">
+                            Keep the landing page that's already configured in
+                            your project.
+                        </p>
+                    </div>
+                </div>
+            </label>
+
+            <label id="defaultLandingPageTemplateLabel">
```

Comment:
So adding new landing pages in the future will require copy-pasting the html snippets here in this file?

Reply:
At the moment. But it's a good point: all of these entries could be dynamically generated based on returned data—which they already get, to drive their visibility.

I had thought about that approach, and at the end of the day wasn't sure how much the entries in this page would fluctuate, to make it worthwhile to drive the content dynamically. We're well positioned to do so though, if we decide to go that route.